src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts

```diff
@@ -105,7 +96,7 @@ export class ZakenWerkvoorraadComponent
     public utilService: UtilService,
     public dialog: MatDialog,
     private identityService: IdentityService,
-    private websocketService: WebsocketService,
+    websocketService: WebsocketService,
```

Comment:
Minor thing but why remove the `private` here?

Reply:
It's a bit of a Typescript quirk, but: when you add a access modifier to a constructor argument, Typescript automatically makes a field of that type on the class and initializes it with the value of the argument. so `this.websocketService = websocketService`. We don't need that anymore: we now only use the websocketService to instantiate an instance of the BatchProcessService.